README.md

```diff
@@ -2,7 +2,7 @@
 
 # peril
 
-Settings for the RxSwiftCommunity organization's [Peril](https://github.com/danger/peril) server. The server receives webhook events from every org repository (new issues and pull requests). We can hook into that in the `issue.ts` and `pr.ts` files, respectively. Usually we only use `pr.ts`.
+Settings for the RxSwiftCommunity organization's [Peril](https://github.com/danger/peril) server. The server receives webhook events from every org repository (new issues and pull requests). We can hook into that in the `org/all-issues.ts` and `org/all-prs.ts` files, respectively. Usually we only use `all-pr.ts`, but Peril affords us a lot of power and flexibility to respond to [pull request actions](https://developer.github.com/v3/activity/events/types/#pullrequestevent) and [issue actions](https://developer.github.com/v3/activity/events/types/#issuesevent).
```

Comment:
One thing I noticed, you wrote `all-pr` and `all-prs` (plural)
Which is right?

Reply:
@ashfurrow one minor comment

Reply:
The plural is what I meant, good catch.